src/main/java/seedu/taassist/model/moduleclass/UniqueModuleClassList.java

```diff
@@ -33,7 +34,7 @@ public class UniqueModuleClassList implements Iterable<ModuleClass> {
      */
     public boolean contains(ModuleClass toCheck) {
         requireNonNull(toCheck);
-        return internalList.stream().anyMatch(toCheck::isSameModuleClass);
+        return internalList.stream().anyMatch(toCheck::equals);
```

Comment:
Okay for now. But this behaviour should be noted when we start implementing additional fields to module class. What does it then mean for two modules to be equal? Note that `equals` method is also used for unit testing. So, perhaps it might be a great idea to revert back to the original design down the line?

Reply:
Yep, good suggestion! We will see how we actually implement the sessions with the module classes and modify accordingly. If you want, you can create an issue to remember this change later on!